backend/src/agents/crew.py

```
@@ -0,0 +1,88 @@
+from crewai import Crew, Agent, Process, Task, LLM
+import os
+from crewai.project import CrewBase, agent, tool, crew, task
```

Ruff (0.12.2)

`crewai.project.tool` imported but unused

Remove unused import: `crewai.project.tool`

(F401)